srcpkgs/dmd2.091/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'dmd2.091'
+pkgname=dmd2.091
+version=2.091.0
+revision=1
+create_wrksrc=yes
+hostmakedepends="which"
+makedepends="dmd2.081 git"
```

Comment:
Shouldn't `dmd` go in hostmakedepends? I see this is how the previous template was doing it, but still.

Reply:
We could set this package as `nocross`, too. No point on building this package for cross.